server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableImpl.java

```diff
@@ -1557,30 +1558,27 @@ public boolean isMajorCompactionQueued() {
    * should be running and none should be able to start.
    */
   public synchronized void close() {
-    synchronized (this) {
```

Comment:
Looked into the way this code works, think we should keep this sync block and remove the sync keyword on the method.  There is no need for the last stmt to be in the sync block, the code in `manager.compactableClosed(` is thread safe and its already called by multiple threads because the lock in this code is per tablet and many tablets can already concurrently call `manager.compactableClosed(`.